### ISSUE TITLE
macOS: alternate character choice writes redundant character

### ISSUE THREAD

**Issue by al-soup** [securingsincity/react-ace#819]
# Problem

Platform: macOS
OS specifics: Catalina 10.15.4

When an alternate (special) character is selected by holding down the key and choosing  it with the respective number, the held down character is written as well as the alternate character.

<img width="285" alt="Screenshot 2020-04-05 at 18 46 41" src="https://user-images.githubusercontent.com/50832516/78504475-d19ba300-776d-11ea-8d99-ef710e47e651.png">

**Comment by nightwing** [comment]
Which version of ace do you use? This was a bug in old versions, but is fixed in the current version.

**Comment by al-soup** [comment]
The screen-shot above is from the react-ace [github demo](http://securingsincity.github.io/react-ace/). Could you please tell me in which version the bug was fixed?

**Comment by nightwing** [comment]
The original bug causing duplication was fixed in 1.4.0, so  this is something else then. Unfortunately i was not able to reproduce the issue on Catalina 10.15.4. Do you see this issue on all browsers or only some? Aso do you see the bug on https://ace.c9.io/ too?

**Comment by al-soup** [comment]
Yes on https://ace.c9.io/ too. You are right, but it seems only to be a problem with Firefox. I tested with Firefox 74.0.1 and Firefox Developer Edition 75.0b12. Chrome and Safari work correctly.
